src/Audit.js

```diff
@@ -2,6 +2,7 @@
 'use strict'
 
 const debug = require('debug')('is-website-vulnerable')
+const argv = require('yargs').argv
```

Comment:
@nopantsmonkey how about we put this argv in the main CLI file where command arguments are received, and then pass it to the Audit class via the audit command with a function argument?